--- a/src/pybind/mgr/dashboard/CMakeLists.txt	2018-08-10 14:50:49.442579472 +0200
+++ b/src/pybind/mgr/dashboard/CMakeLists.txt	2018-08-10 14:24:00.474147421 +0200
@@ -27,7 +27,7 @@
 
 add_custom_command(
   OUTPUT "${CMAKE_SOURCE_DIR}/src/pybind/mgr/dashboard/frontend/node_modules"
-  COMMAND . ${mgr-dashboard-nodeenv}/bin/activate && npm install && deactivate
+  COMMAND npm install
   DEPENDS frontend/package.json
   WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}/src/pybind/mgr/dashboard/frontend
   COMMENT "dashboard frontend dependencies are being installed"
@@ -64,7 +64,7 @@
 
 add_custom_command(
   OUTPUT "${CMAKE_SOURCE_DIR}/src/pybind/mgr/dashboard/frontend/dist"
-  COMMAND . ${mgr-dashboard-nodeenv}/bin/activate && ${npm_command} && deactivate
+  COMMAND ${npm_command}
   DEPENDS ${frontend_src} frontend/node_modules
   WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}/src/pybind/mgr/dashboard/frontend
   COMMENT "dashboard frontend is being created"
